packages/rest/src/rest.server.ts

```diff
@@ -425,7 +426,7 @@ export class RestServer extends Context implements Server, HttpServerLike {
     );
 
     specForm = specForm ?? {version: '3.0.0', format: 'json'};
-    const specObj = this.getApiSpec(requestContext);
+    const specObj = ConsolidateSchemaObjects(this.getApiSpec(requestContext));
```

Comment:
Please perform the consolidation from inside `this.getApiSpec()`. We should provide the same spec regardless of whether it was obtained via REST API or by calling `server.getApiSpec()`.